src/status_im2/contexts/communities/home/style.cljs

```diff
@@ -42,3 +43,34 @@
    :left        0
    :padding-top top})
 
+;;;; CARD ANIMATION
+(def card-bottom-override {:margin-bottom 16}) ; Original 8 + 8 from tabs top padding
+(def card-height (+ 56 16)) ; Card height + its vertical margins
+(def card-total-height (+ card-height 8)) ; added 8 from tabs top padding
+(def card-opacity-factor (/ 100 card-height 100))
+
+(defn- value-in-range
```

Comment:
To me, general numeric functions should be moved to a utility namespace such as `utils.number` and ideally unit tested (although okay, this one is trivial).

I found another implementation in the code doing the exact same thing:

```clojure
;; status-im2.contexts.chat.composer.utils
(defn bounded-val
  [val min-val max-val]
  (max min-val (min val max-val)))
```

This is the problem I have with the reasoning that functions with only one usage should be placed where they're used (a past discussion here in status-mobile). This practice doesn't promote reusability.

The question I try to answer is: "which layer of the system is concerned with this function's responsibility?" and the answer is clear to me, not the UI.

I also prefer the simpler function signature without the need for destructuring. You'd just need to change your calls to be `(value-in-range new-opacity 0 1)`.

Reply:
Ok, I will move the function to `utils.number` and remove the duplicates

Reply:
done :+1: